.ado/apple-integration.yml

```diff
@@ -37,6 +37,8 @@ jobs:
           echo "##vso[task.setvariable variable=package_version]$(cat package.json | jq .version | awk '{ print substr($0, 2, length($0) - 2) }')"
           echo "##vso[task.setvariable variable=react_version]$(cat package.json | jq .peerDependencies.react)"
           echo "##vso[task.setvariable variable=rncli_version]$(cat package.json | jq '.dependencies."@react-native-community/cli"')"
+          echo "##vso[task.setvariable variable=rncli_android_version]$(cat package.json | jq '.dependencies."@react-native-community/cli-platform-android"')"
+          echo "##vso[task.setvariable variable=rncli_ios_version]$(cat package.json | jq '.dependencies."@react-native-community/cli-platform-ios"')"
```

Comment:
@tido64 fyi. I needed to make this change because cli is at 7.0.3 while cli-platform-ios/android are at 7.0.1. So the script modified the dependencies below on line 67/68 incorrectly.